src/statmodels.jl

```diff
@@ -200,6 +200,13 @@ const adjr² = adjr2
 abstract type RegressionModel <: StatisticalModel end
 
 """
+    coefnames(obj::RegressionModel)
```

Comment:
Should allow any `StatisticalModel`, just like `coef`.

Reply:
Made it inherit from `StatisticalModel`. Makes sense since `coef` is at the `StatisticalModel` level.